### PR TITLE
[Localization] Locales punctuation fix in descriptions in (en/es/de/it/fr/ko/pt/zh) modifier-type.ts, and other German locales punctuation fixes

### PR DESCRIPTION
**Pull request by InnocentGameDev**
## Why am I doing these changes?
Ok, so I'll start with this considering the nature of the change. I made some other localization changes in another PR and was getting suggestions that included a "." at the end of the description of items. 95% of English item descriptions didn't end with punctuation and almost all of the Spanish, Italian and German item descriptions didn't have punctuation either. French was at 50/50 split.

So two goals are achieved here:
1- Consistency across the board.
2- Said consistency replicates the S/V item descriptions.

## What are the changes?
For now, not all modifier-type.ts are changed, but I wanted to make the PR already in case anyone is working on this as well.

- [X] English punctuation fix in modifier-type.ts
- [X] Spanish punctuation fix in modifier-type.ts
- [x] German punctuation fix in modifier-type.ts and other places too, thanks @CodeTappert 
- [x] French punctuation fix in modifier-type.ts
- [x] Italian punctuation fix in modifier-type.ts
- [x] PT_BR punctuation fix in modifier-type.ts
- [x] Korean punctuation fix in modifier-type.ts and Berry Pouch locale fix, thanks @MrWaterT 
- [x] Chinese_CN punctuation fix
- [x] Chinese_TW punctuation fix

## What did change?
Descriptions in locales/lang/modifier-type.ts got proper punctuation when applicable.

The commit also contains toxic/flame orb description in English being fixed, done wrong by me and fixed here -- wasn't the one in the main games, and also a Wind Power translation in DE which was missing -- this last one is by @CodeTappert.

I can confirm that Traditional Chinese descriptions in the main games use "。", but it would be nice to have a native speaker review it just in case they also have some sort of suffix that needs to be added to the sentence as well. ChatGPT says the translations are still well written in 6-7 random examples I tested.

This PR https://github.com/pagefaultgames/pokerogue/pull/1472 missed the number change in the korean locale for newest Berry Pouch change, so that's added too by @MrWaterT 

## How to test the changes?
Sentences should now show ".", -- or  "。" for the Chinese translations --when showing an item's description.

## Screenshots

image

post

## Checklist
- [X] There is no overlap with another PR? There have been many merge conflicts throughout the approval of this which have been merged.

- [X] The PR is self-contained and cannot be split into smaller PRs? It bundles some minor locale changes, but it's 99% localisation.

- [X] Have I provided a clear explanation of the changes? I believe so.

- [x] Have I tested the changes (manually)? Yes.
    - [X] Are all unit tests still passing? (`npm run test`) Yes.
    
    
- [X] Are the changes visual? Yes.
  - [x] Have I provided screenshots/videos of the changes? Yes.

### CODE REVIEW

**Comment by InnocentGameDev** [comment]
The Berry Pouch numbers change made it to live before this so I had to re-add the "."s to resolve the conflict

File: src/locales/ko/modifier-type.ts [hunk 1]
```diff
@@ -178,36 +178,36 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
 
     "MULTI_LENS": { name: "멀티렌즈" },
 
-    "HEALING_CHARM": { name: "치유의부적", description: "HP를 회복하는 기술을 썼을 때 효율이 10% 증가 (부활 제외)" },
-    "CANDY_JAR": { name: "사탕단지", description: "이상한사탕 종류의 아이템이 올려주는 레벨 1 증가" },
+    "HEALING_CHARM": { name: "치유의부적", description: "HP를 회복하는 기술을 썼을 때 효율이 10% 증가한다 (부활 제외)." },
+    "CANDY_JAR": { name: "사탕단지", description: "이상한사탕 종류의 아이템이 올려주는 레벨이 1 증가한다." },
 
-    "BERRY_POUCH": { name: "열매주머니", description: "사용한 나무열매가 소모되지 않을 확률 33% 추가" },
+    "BERRY_POUCH": { name: "열매주머니", description: "사용한 나무열매가 소모되지 않을 확률이 33% 추가된다." },
```
Comment (MrWaterT) [inline src/locales/ko/modifier-type.ts:201]:
```suggestion
    "BERRY_POUCH": { name: "열매주머니", description: "사용한 나무열매가 소모되지 않을 확률이 30% 추가된다." },
```

Comment (InnocentGameDev) [reply src/locales/ko/modifier-type.ts:201]: Didn't see korean didn't show up at all in the merge conflict. Other PR author must've not seen it. Added

**Review by CodeTappert — APPROVED** [review]

File: src/locales/zh_CN/modifier-type.ts [hunk 2]
```diff
@@ -178,64 +178,64 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
+    "KINGS_ROCK": { name: "王者之证", description: "携带该道具的宝可梦使用任意原本不会造成\n畏缩状态的攻击招式并造成伤害时，有\n10%几率使目标陷入畏缩状态。" },
 
-    "LEFTOVERS": { name: "吃剩的东西", description: "携带该道具的宝可梦在每个回合结束时恢复\n最大HP的1/16" },
-    "SHELL_BELL": { name: "贝壳之铃", description: "携带该道具的宝可梦在攻击对方成功造成伤\n害时，携带者的HP会恢复其所造成伤害\n的1/8" },
+    "LEFTOVERS": { name: "吃剩的东西", description: "携带该道具的宝可梦在每个回合结束时恢复\n最大HP的1/16。" },
+    "SHELL_BELL": { name: "贝壳之铃", description: "携带该道具的宝可梦在攻击对方成功造成伤\n害时，携带者的HP会恢复其所造成伤害\n的1/8。" },
 
-    "TOXIC_ORB": { name: "Toxic Orb", description: "It's a bizarre orb that exudes toxins when touched and will badly poison the holder during battle" },
-    "FLAME_ORB": { name: "Flame Orb", description: "It's a bizarre orb that gives off heat when touched and will affect the holder with a burn during battle" },
+    "TOXIC_ORB": { name: "Toxic Orb", description: "이 도구를 지닌 포켓몬은 턴이 끝나는 시점에 상태이상에 걸리지 않았다면 맹독 상태가 된다." },
```
Comment (brain-frog) [inline src/locales/zh_CN/modifier-type.ts:212]: these should use the same period symbol as the rest

**Review by brain-frog — CHANGES_REQUESTED** [review]
small change to keep consistency

File: src/locales/ko/modifier-type.ts [hunk 3]
```diff
@@ -17,122 +17,122 @@ export const modifierType: ModifierTypeTranslationEntries = {
...
+      description: "기술의 명중률이 {{accuracyAmount}} 증가한다 (최대 100).",
     },
     "PokemonMultiHitModifierType": {
-      description: "공격이 가진 갯수에 따라 60/75/82.5%의 위력으로 한번 더 명중",
+      description: "기술이 가진 갯수에 따라 60/75/82.5%의 위력으로 한번 더 공격한다.",
     },
     "TmModifierType": {
       name: "No.{{moveId}} {{moveName}}",
-      description: "포켓몬에게 {{moveName}}[[를]] 가르침",
+      description: "포켓몬에게 {{moveName}}[[를]] 가르침친다.",
```
Comment (MrWaterT) [inline src/locales/ko/modifier-type.ts:100]:
```suggestion
      description: "포켓몬에게 {{moveName}}[[를]] 가르친다.",
```

File: src/locales/ko/modifier-type.ts [hunk 4]
```diff
@@ -217,25 +217,25 @@ export const modifierType: ModifierTypeTranslationEntries = {
     "SHINY_CHARM": { name: "빛나는부적", description: "야생 포켓몬이 색이 다른 포켓몬으로 등장할 확률을 급격히 증가" },
     "ABILITY_CHARM": { name: "특성부적", description: "야생 포켓몬이 숨겨진 특성을 가지고 등장할 확률을 급격히 증가" },
```
Comment (MrWaterT) [inline src/locales/ko/modifier-type.ts:218]:
```suggestion
    "SHINY_CHARM": { name: "빛나는부적", description: "야생 포켓몬이 색이 다른 포켓몬으로 등장할 확률을 급격히 높인다." },
    "ABILITY_CHARM": { name: "특성부적", description: "야생 포켓몬이 숨겨진 특성을 가지고 등장할 확률을 급격히 높인다." },
```

**Review by MrWaterT — COMMENTED** [review]
Edit typo (maybe) resolve conflic.
Add punctuations to missing.

toxic orb, flame orb, berry pouch has no problem in ko.

**Review by brain-frog — APPROVED** [review]

**Pull request merged**